tests/primer/primer_tool.py

```diff
@@ -88,6 +100,35 @@ def _handle_prepare_command(self) -> None:
             ) as f:
                 f.write(commit_string)
 
+    def _handle_run_command(self) -> None:
+        if Path(PRIMER_DIRECTORY / f"output_{self.config.type}.txt").exists():
+            os.remove(PRIMER_DIRECTORY / f"output_{self.config.type}.txt")
+
+        packages: dict[str, list[dict[str, str | int]]] = {}
+
+        for package, data in self.packages.items():
+            output = self._lint_package(data)
+            packages[package] = output
+            print(f"Successfully primed {package}.")
+
+        with open(
+            PRIMER_DIRECTORY / f"output_{self.config.type}.txt", "a", encoding="utf-8"
```

Comment:
If multiple PRs are each running the primer step, does this mean they will append their results to each other?

Reply:
No, they don't share their environments.